### PR TITLE
improve mds error log for site extension install/uninstall

### PR DESCRIPTION
**Pull request by shrimpy**

### CODE REVIEW

File: Kudu.Core/Tracing/IAnalytics.cs [hunk 1]
```diff
@@ -11,7 +8,9 @@ public interface IAnalytics
 
         void JobStarted(string jobName, string scriptExtension, string jobType, string siteMode, string error);
 
-        void UnexpectedException(Exception ex, bool trace = true);
+        void UnexpectedException(Exception exception, bool trace = true);
+
+        void UnexpectedException(Exception ex, string method, string path, string result, string message, bool trace = true);
```
Comment (suwatch) [inline Kudu.Core/Tracing/IAnalytics.cs]: ex or exception, do pick one.

Comment (shrimpy) [inline Kudu.Core/Tracing/IAnalytics.cs]: will rename to "ex". thanks for noticing that.

**Comment by suwatch** [comment]
Do share out MDS.  Other than that, :shipit:

**Comment by shrimpy** [comment]
yes. just sent out. thanks for the reminding.

**Pull request merged**